rockcraft/oci.py

```diff
@@ -320,15 +339,62 @@ def _add_layer_into_image(image_path: Path, archived_content: Path, **kwargs) ->
     _process_run(cmd + ["--history.created_by", " ".join(cmd)])
 
 
-def _archive_layer(layer_path: Path, temp_tar_file: Path) -> None:
+def _archive_layer(
+    layer_path: Path, temp_tar_file: Path, lower_rootfs: Optional[Path] = None
+) -> None:
     """Prepare new OCI layer by archiving its content into tar file.
 
     :param layer_path: path to the content to be archived into a layer
     :param temp_tar_file: path to the temporary tar fail holding the archived content
```

Comment:
I know you didn't touch this line but I saw a typo anyway

```suggestion
    :param temp_tar_file: path to the temporary tar file holding the archived content
```